src/plugins/pol/encoding.h

```diff
@@ -149,19 +149,18 @@ inline std::basic_string<target_char> convert(string_const_iterator<source_char>
     char *inbuf = reinterpret_cast<char *>(const_cast<source_char *>(&*begin));
     size_t inbytesLeft = std::distance(begin, end) * sizeof(source_char);
 
-    auto temp = std::make_unique<std::array<char, 512>>();
-    char *outbuf = temp->data();
+    auto temp = std::make_unique<std::array<target_char, 512>>();
```

Comment:
What is `target_char` here where it is defined?

Reply:
this is a template variable declared in "src/plugins/pol/encoding.h:143:20". "target_char" is the character type "std::basic_string<target_char>", which will be converted from "std::basic_string<source_char>".